packages/table/src/Row/Row.stories.tsx

```diff
@@ -228,4 +238,136 @@ DisabledClickableRows.args = {
   disabled: true,
 };
 
+export const DisabledSelectableRows: StoryFn<
```

Comment:
I think this is an unrelated issue, but should you still be able to select/unselect all when the rows are disabled? https://642700aa3e49e32bdbf0b0fc-jevezbjkag.chromatic.com/?path=/story/components-table-row--disabled-selectable-rows&args=disabled:!true

Reply:
Created LG-3612 to explore this!